CarSpaceDev/mobile_frontend
# Geolocator updates

### Changes 
Login redirects directly to MapScreen
MapScreen is Google Maps with a location stream listener

### Testing Steps
-check in constants if the API url is correct 
-run app
-log in with google
-you should see the pin for your current location and moving around should show it moving

[Setup geolocator stream](https://app.gitkraken.com/glo/board/5ef861cff951d500124d7f85/card/5ef8622042d430001111e0ff)
[Detect and display user's location in the map](https://app.gitkraken.com/glo/board/5ef861cff951d500124d7f85/card/5ef8c4f384d5570011507dc6)
